extension/content.css

```diff
@@ -242,18 +242,46 @@ forked a repo
 remove padding from body of simple news alerts in Dashboard since some will be hidden
 we will add it back on for the simple news alerts we decide to show
 */
+/* - Disabled because these rules were overwriting other padding rules, breaking the layout. -
```

Comment:
Disabling this introduces a bug. We are hiding notifications when creating a branch, but we also need to remove the padding so there's no extra spacing. See screenshot:

![screen shot 2016-05-15 at 10 01 34 pm](https://cloud.githubusercontent.com/assets/737065/15278600/c55b185e-1ae8-11e6-86b8-b54e36df08ff.png)

Also, if you find a solution that doesn't include this CSS (or other CSS), you should delete it not comment it out.